extensions/fetch/lib.rs

```diff
@@ -524,6 +546,15 @@ pub fn create_http_client(
     builder = builder.add_root_certificate(cert);
   }
 
+  if let Some(proxy) = proxy {
+    let mut reqwest_proxy = reqwest::Proxy::all(&proxy.url)?;
```

Comment:
The proxy URL should go through a network permission check here.

Reply:
Or more accurately in the op_create_http_client

Reply:
@kitsonk @lucacasonato Thank you for your comments.

I fixed to check a network permission.